src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -68,11 +69,12 @@ public class ConsumerConfiguration implements JsonSerializable {
     protected final Duration maxExpires;
     protected final Duration inactiveThreshold;
     protected final Long startSeq; // server side this is unsigned
-    protected final Long maxDeliver;
+    protected final Integer maxDeliver;
     protected final Long rateLimit; // server side this is unsigned
-    protected final Long maxAckPending;
-    protected final Long maxPullWaiting;
-    protected final Long maxBatch;
+    protected final Integer maxAckPending;
+    protected final Integer maxPullWaiting;
+    protected final Integer maxBatch;
+    protected final Integer maxBytes;
```

Comment:
After reviewing the server code I realized that the fields were go int, which are 32 bit signed values aka java int. I have only changed the internal variable type, getters and builders were left the same.